data_analysis/notebooks/businesses.ipynb

```diff
@@ -3,9 +3,20 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "id": "bf058d3d",
```

Comment:
Can you add Colorado Nonprofit Association to the data sources section of README.txt?